### ISSUE TITLE
shadow_getEthHeaderByNumber not working

### ISSUE THREAD

**Issue by HackFisher** [darwinia-network/darwinia.js#37]
```
promote:~ denny$ curl -d '{"method":"shadow_getEthHeaderByNumber","params":{"block_num": 10000000}, "id": 0}' https://shadow.darwinia.network
{"id":0,"result":null,"error":"The requested block number is too low, only support blocks heigher than 0"}
```

**Comment by clearloop** [comment]
> ```
> promote:~ denny$ curl -d '{"method":"shadow_getEthHeaderByNumber","params":{"block_num": 10000000}, "id": 0}' https://shadow.darwinia.network
> {"id":0,"result":null,"error":"The requested block number is too low, only support blocks heigher than 0"}
> ```

This issue belongs to `darwinia.go`, will be fixed after merging [#17][0]

[0]: https://github.com/darwinia-network/darwinia.go/pull/17

**Comment by clearloop** [comment]
Fixed in #17, and now `dj` is using HTTP requests. for example:

```
$ curl https://shadow.darwinia.network/block/10000000
```
